android/java/org/chromium/chrome/browser/notifications/BraveNotificationPlatformBridge.java

```diff
@@ -77,18 +77,19 @@ protected NotificationBuilderBase prepareNotificationBuilder(String notification
             vibrationPattern = EMPTY_VIBRATION_PATTERN;
         }
 
-        return super.prepareNotificationBuilder(notificationId, notificationType, origin, scopeUrl,
-                profileId, incognito, vibrateEnabled, title, body, image, icon, badge,
-                vibrationPattern, timestamp, renotify, silent, actions, webApkPackage);
-    }
+        NotificationBuilderBase result =
+                super.prepareNotificationBuilder(notificationId, notificationType, origin, scopeUrl,
+                        profileId, incognito, vibrateEnabled, title, body, image, icon, badge,
+                        vibrationPattern, timestamp, renotify, silent, actions, webApkPackage);
 
-    @Override
-    protected NotificationBuilderBase createNotificationBuilder(Context context, boolean hasImage) {
-        if (mNotificationType == NotificationType.BRAVE_ADS) {
-            // TODO(jocelyn): Remove setPriority here since we already set the
-            // importance of Ads notification channel to IMPORTANCE_HIGH?
-            return new BraveAdsNotificationBuilder(context).setPriority(Notification.PRIORITY_HIGH);
+        assert result
+                instanceof BraveAdsNotificationBuilder
```

Comment:
As I understand, now `BraveAdsNotificationBuilder` builds all Brave notifications, not only Brave Ads notifications. Should we rename it to something like `BraveNotificationBuilder`?

Reply:
Yep, this is reasonable renaming, should be quick so I'll do it here.

Reply:
Fixed in 41994953eaabbee4fa5e03f54267c4396d34828b